docs/part-7.md

```diff
@@ -0,0 +1,267 @@
+---
+id: part-7
+title: Part 7. Functional Programming Introduction
+---
+
+# Functional Programming
+
+Contributors:
+
+- Sergi Dote
+
+## Introduction
+
+Functional programming is a paradigm that puts the attention on functions. But this definition can be a little confusing since developers are coding using functions and methods and this doesn't mean they are programming following the FP paradigm.
```

Comment:
issue: Correct the article by removing *the*.
```suggestion
Functional programming is a paradigm that puts attention on functions. But this definition can be a little confusing since developers are coding using functions and methods and this doesn't mean they are programming following the FP paradigm.
```

Reply:
Solved!